deploy/charts/cert-manager/templates/webhook-deployment.yaml

```diff
@@ -61,10 +61,18 @@ spec:
           {{- if .Values.global.logLevel }}
           - --v={{ .Values.global.logLevel }}
           {{- end }}
+          {{- if .Values.webhook.config }}
+          - --config=/var/cert-manager/config/config.yaml
+          {{- end }}
+          {{- $config := default .Values.webhook.config "" }}
+          {{ if not $config.securePort -}}
           - --secure-port={{ .Values.webhook.securePort }}
+          {{- end }}
```

Comment:
I noticed that if I set `securePort` in the (helm deployed) config file I still have to also set the securePort value, otherwise the container port mapping  in the pod template is wrong (below)

```
global:
  logLevel: 3
installCRDs: true
webhook:
  securePort: 8443
  config:
    apiVersion: webhook.config.cert-manager.io/v1alpha1
    kind: WebhookConfiguration
    securePort: 8443
    enablePprof: true

```

I don't really know how to get round that though.

Reply:
> otherwise the container port mapping in the pod template is wrong (below)

ahhhh this is a good point that I missed! :)

Reply:
... also highlights an area where we do not perform automated testing 👀